plugin/trino-geospatial/src/main/java/io/trino/plugin/geospatial/BingTileFunctions.java

```diff
@@ -575,8 +576,8 @@ private static void appendIntersectingSubtiles(
     private static Point tileXYToLatitudeLongitude(int tileX, int tileY, int zoomLevel)
     {
         long mapSize = mapSize(zoomLevel);
-        double x = (clip(tileX * TILE_PIXELS, 0, mapSize) / mapSize) - 0.5;
-        double y = 0.5 - (clip(tileY * TILE_PIXELS, 0, mapSize) / mapSize);
+        double x = (clamp(tileX * TILE_PIXELS, 0d, mapSize) / mapSize) - 0.5;
```

Comment:
we're clamping integer numbers, so `d` in `0d` is redundant.
cast clamp result explicitly to double